src/claim/authorize-ksign/authorize-ksign.js

```diff
@@ -0,0 +1,72 @@
+const Claim = require('../claim');
+const utils = require('../../utils');
+
+/**
```

Comment:
Please, short description of the class and the parameters go to the constructor 
(http://usejsdoc.org/howto-es2015-classes.html)